recipes/crms2map/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "crms2map" %}
+{% set version = "1.0" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name |lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jinikeda/{{ name |lower }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 7c718dbe50dd3fe7cb2cf294418e846028126ed6a4f9c3960db92b7cfce3ae53
+
+build:
+  noarch: python
+  entry_points:
+    - CRMS2Map = src.click_main:click_main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+  host:
+    - python {{ python_min }} 
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - gdal
+    - numpy
+    - pandas
+    - geopandas
+    - rasterio
+    - scipy
+    - matplotlib-base
+    - seaborn
+    - basemap
+    - rioxarray
+    - click
+
+test:
+  imports:
+    - src.CRMS_general_functions
```

Comment:
This doesn't look right, packages should not claim the `src` folder.

Reply:
Thank you for your feedback regarding the src folder in the meta.yaml file. I slightly modified meta.yaml. The test codes are designed for simple functional tests for individual components rather than a comprehensive package test.

Reply:
Please address this. This is a issue with the upstream package but as you are upstream, you should fix that yourself.

Reply:
Thank you for your comment. Based on your comment, I fixed the code. Thank you.